server/planning/assignments/module.py

```diff
@@ -34,3 +35,34 @@
     ),
     elastic=ElasticResourceConfig(),
 )
+
+delivery_resource_config = ResourceConfig(
+    name="delivery",
+    data_class=DeliveryResourceModel,
+    service=DeliveryAsyncService,
+    mongo=MongoResourceConfig(
+        indexes=[
+            MongoIndexOptions(
+                name="planning_id_1",
+                keys=[("planning_id", 1)],
+                background=True,
+            ),
+            MongoIndexOptions(
+                name="assignment_id_1",
+                keys=[("assignment_id", 1)],
+                background=True,
+            ),
+            MongoIndexOptions(
+                name="coverage_id_1",
+                keys=[("coverage_id", 1)],
+                background=True,
+            ),
+            MongoIndexOptions(
+                name="item_id_1",
+                keys=[("item_id", 1)],
+                background=True,
+            ),
+        ],
+    ),
+    elastic=ElasticResourceConfig(),
```

Comment:
Elasticsearch is not enabled for this resource (we only use Mongo for this one).

One way you can tell if Elastic is enabled for a resource, in the old resource class we have a `datasource` dict attribute that has `"search_backend": "elastic"` defined ([example](https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/assignments/assignments.py#L1531)).

If the `search_backend` attribute is not defined, or not set to `elastic`, then Elasticsearch not is enabled for that resource